test/mocks/network/mocks.h

```diff
@@ -164,6 +164,7 @@ class MockListenerFilter : public ListenerFilter {
   MockListenerFilter();
   ~MockListenerFilter() override;
 
+  MOCK_METHOD0(die_, void());
```

Comment:
s/die_/destroy which is what we typically name this method

Reply:
Updating...
Looks like `destroy_` w/ underscore is preferred since it is added in MockClass